live/index.html

```diff
@@ -0,0 +1,747 @@
+
+					<p>Your challenge is to create a solution that enables SAP employees to capture their environmental impact while commuting,
+						at work, or on business trips. You are free to focus on either implicit (through sensors, usage data, etc.) or explicit
+						(e.g., logging shared car rides with colleagues, logging elevator rides) data collection. For either case, the manual
+						effort for employees should be kept to a minimum, but still yield complete and meaningful data.</p>
+
+					<h3>#2 Offset Your Browsing Emissions</h3>
+
+					<p>Beyond the immediate cost of energy consumed by your Smartphone, Notebook, TV, etc., there’s a hidden impact on the environment that originates from the servers used to serve your request.</p>
+					<p>Your task is to estimate the CO2 usage caused by using popular web sites (google search cost more than viewing a static html page etc.) and allow users to easily offset their emission (e.g., by purchasing CO2 compensation certificates, donating money to environmental organizations, etc.). </p>
+					<p>A different aspect of this topic is to replace the emission estimate with concrete numbers based on software components involved in serving user requests. Your task is provide a solution to estimating the CO2 footprint of different IT services involved in serving user requests. </p>
+
+					<!--<h3>Prize:</h3>
+
+					<p>4 x TBA</p>-->
+
+					<p>
+						<strong>Important note: </strong>To be considered for the SAP prize, your project needs to be published under the
+						<a href="https://opensource.org/licenses/MIT"
+						    target="_blank">MIT License</a>. For that, you can simply include a file named LICENSE containing the license text in your repository / project folder.</p>
+				</div>
+			</div>
+
+			<div class="row sponsor-information">
+				<div class="col-md-4">
+					<img src="./live_files/wwf_noWhiteBg.png" class="img-responsive"> </div>
+
+				<div class="col-md-8">
+					<h2>WWF</h2>
+
+					<h3>#1 How can we make climate change more tangible?</h3>
+
+					<p>The climate crisis is no far off phenomena, it has reached our homes. Its consequences, for example extreme weather
+						events like heavy rain and heatwaves, is affecting the nature just outside our front doors. Growth patterns of plants
+						and crops change just like behavioural patterns of animals such as migratory birds. Some plants and animals cannot
+						cope with the changing environment, they disappear altogether.</p>
+
+					<p>We want people not only to open their eyes to these changes. We want them to share their observations with others, building
+						a network of locally engaged citizens who connect over the discussion of climate change in their region. Ideally, we
+						want them to become active in the fight against climate change.</p>
+
+					<h3>#2 How can we make climate change more tangible? pt. 2</h3>
```

Comment:
This challenge is apparently wrong. As you can see both challenges have the same title, meaning that there are two options provided by WWF for the first challenge. I do not know which one to take, but @annkatrinkuessner should know. The second challenge (#2) is found in the known shared document.

Reply:
right, back then the 2nd was not yet in the doc; is updated now

for challenge #1, both options are included now. can you confirm this is correct? @annkatrinkuessner